setup/conda/conda_env.yml

```diff
@@ -0,0 +1,14 @@
+name: sc_env
```

Comment:
nit: maybe simply name this `environment.yml` to follow conda convention https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually